start

```diff
@@ -116,6 +123,10 @@ function process_args() {
 		exit 1
 	esac
 
+	if [ "$RANDOMIZE_NETWORK_PASSPHRASE" = true ]; then
+		NETWORK_PASSPHRASE+=$RANDOM
```

Comment:
❗ Could you use a separator between the network passphrase and the random data? For example, in the passphrases they seem to use ` ; `, could you add that before the random value so that it isn't tacked on with the year?